### DISCUSSION TITLE
[question] AA hooks with a Web3Auth signer

### DISCUSSION THREAD

**Discussion by bmarcot** [alchemyplatform/aa-sdk#853] [Q&A] [answered]
Is is possible to use the  `aa-alchemy`  hooks with a Web3Auth (modal) signer and not the default Turnkey signer?

**Comment by avarobinson** [comment, marked as answer]
Hi @bmarcot.

Right now, no, you can only use the`aa-alchemy` React Hooks when using the Alchemy Signer. Using the [Alchemy Signer](https://accountkit.alchemy.com/signers/alchemy-signer/introduction#creating-an-alchemy-signer) will greatly simplify your development and costs!

To use a third-party signer, please follow the various [integration guides](https://accountkit.alchemy.com/signers/guides/web3auth) found in our docs.
